src/test/java/com/bhoruka/bloodbank/TestConstants.java

```diff
@@ -0,0 +1,45 @@
+package com.bhoruka.bloodbank;
+
+import com.bhoruka.bloodbank.model.CampModel;
+import com.bhoruka.bloodbank.model.request.CreateCampRequest;
+import com.bhoruka.bloodbank.model.response.CreateCampResponse;
+
+import java.util.Date;
+
+import org.springframework.http.HttpStatus;
+
+public final class TestConstants {
+
+    public static final Date TEST_DATE = new Date();
+
+    public static final String TEST_CAMP_ID = "TestCamp1";
+
+    public static final String TEST_PARTNER_ID = "ABC";
+
+    private static final String CAMP_CREATE_SUCCESS_MESSAGE = "Camp created successfully.";
```

Comment:
Please refer to the previous comment regarding putting this particular constant in a different interface/class rather than putting it in every class that might use it

Reply:
This won't be reused in other places as stated previously.